email-auditing-api.yaml

```diff
@@ -0,0 +1,253 @@
+openapi: 3.0.1
+info:
+  description: |
+        Email Auditing API allows for downloading Invitation, Reminder, and/or Feedback (Rapid) response emails in a compressed file for specific Experience programs and Ad Hoc surveys within a defined period of time. By retaining the last version of these emails for up to 30 days, companies can manage their communication policies and comply with standard industry regulations.
+  title: Email Auditing API
+  version: 0.0.2
```

Comment:
Given the `v0` here, is this an EAP API?

Reply:
Yes